docs/core/compatibility/6.0.md

```diff
@@ -24,6 +24,7 @@ If you're migrating an app to .NET 6, the breaking changes listed here might aff
 | [Changed MessagePack library in @microsoft/signalr-protocol-msgpack](aspnet-core/6.0/messagepack-library-change.md) | ❌ | ✔️ |  |
 | [ClientCertificate property doesn't trigger renegotiation for HttpSys](aspnet-core/6.0/clientcertificate-doesnt-trigger-renegotiation.md) | ✔️ | ❌ |  |
 | [EndpointName metadata not set automatically](aspnet-core/6.0/endpointname-metadata.md) | ✔️ | ❌ | RC 2 |
+| [Identity: Default Bootstrap version of UI changed](aspnet-core/6.0/identity-bootstrap4-to-5.md) | ✔️ | ✔️  |  |
```

Comment:
This is the only change that's both source- and binary-compatible, but that's make me think it might not be classed as a "breaking change" in that case. Perhaps it's just a special case, as it makes the UI look bad, but actually it still compiles and runs.